src/ontology/uberon-edit.obo

```diff
@@ -118700,7 +118700,7 @@ xref: SCTID:304572005
 xref: UMLS:C1704302 {source="ncithesaurus:Bicuspid_Tooth"}
 xref: Wikipedia:Premolar
 is_a: UBERON:0013164 ! molariform tooth
-relationship: has_part UBERON:0006844 {editor_note="cardinality not allowed with transitivity in OWL", min_count="2", taxon_notes="some premolars may have 3 cusps"} ! cusp of tooth
+relationship: has_part UBERON:0006844 {min_count="2", comment="TAXON NOTE some premolars may have 3 cusps", comment="cardinality not allowed with transitivity in OWL"} ! cusp of tooth
```

Comment:
I have a lingering doubt about whether we are allowed to use more than one comment here. The OBO specification clearly states (section 4.4, [Tag Cardinality Constraints](http://owlcollab.github.io/oboformat/doc/obo-syntax.html#cardinality)) that we can only have one comment *in a Term frame*, but it says nothing about what’s permitted in trailing qualifier blocks.

But the OBO parser does not complain about the presence of two comments and the OBO writer is able to roundtrip the file without any hiccup, so I am going to assume this is perfectly fine.

Reply:
Yes, I agree.